test/test_euler.py

```diff
@@ -738,6 +746,21 @@ def write_soln(write_status=True):
     def rhs(t, q):
         return inviscid_operator(discr, eos=eos, boundaries=boundaries, q=q, t=t)
 
+    filter_order = 8
+    eta = .5
+    alpha = -1.0*np.log(np.finfo(float).eps)
+    nummodes = int(1)
+    for i in range(dim):
+        nummodes *= int(order + dim + 1)
+    nummodes /= math.factorial(int(dim))
```

Comment:
Reference? Would it be helpful to stick this in a utility function?

Reply:
Utility?  Nah I don't think, but I defer to your judgement.